common/src/main/java/com/wynntils/screens/maps/PoiManagementScreen.java

```diff
@@ -0,0 +1,253 @@
+                        43,
+                        CommonColors.WHITE,
+                        HorizontalAlignment.Left,
+                        VerticalAlignment.Top,
+                        TextShadow.NORMAL);
+
+        FontRenderer.getInstance()
+                .renderText(
+                        poseStack,
+                        I18n.get("screens.wynntils.poiManagementGui.name"),
+                        this.width / 2f - 130,
+                        43,
+                        CommonColors.WHITE,
+                        HorizontalAlignment.Left,
+                        VerticalAlignment.Top,
+                        TextShadow.NORMAL);
+
+        FontRenderer.getInstance()
+                .renderText(
+                        poseStack,
+                        "X",
+                        this.width / 2f - 15,
+                        43,
+                        CommonColors.WHITE,
+                        HorizontalAlignment.Left,
+                        VerticalAlignment.Top,
+                        TextShadow.NORMAL);
+
+        FontRenderer.getInstance()
+                .renderText(
+                        poseStack,
+                       "Y",
+                        this.width / 2f + 40,
+                        43,
+                        CommonColors.WHITE,
+                        HorizontalAlignment.Left,
+                        VerticalAlignment.Top,
+                        TextShadow.NORMAL);
+
+        FontRenderer.getInstance()
+                .renderText(
+                        poseStack,
+                        "Z",
+                        this.width / 2f + 80,
+                        43,
+                        CommonColors.WHITE,
+                        HorizontalAlignment.Left,
+                        VerticalAlignment.Top,
+                        TextShadow.NORMAL);
+
+        RenderUtils.drawRect(
+                poseStack,
+                CommonColors.WHITE,
+                this.width/2 - 165,
+                52,
+                0,
+                355,
+                1);
+
+        poiManagerWidgets.forEach(widget -> widget.render(poseStack, mouseX, mouseY, partialTick));
+    }
+
+    @Override
+    public boolean mouseScrolled(double mouseX, double mouseY, double delta) {
+        if (delta < 0.0 && nextButton.active) {
+            nextPage();
+        } else if (delta > 0.0 && previousButton.active) {
+            previousPage();
+        }
+
+        return true;
+    }
+
+    public void populatePois() {
+        for (AbstractWidget widget : poiManagerWidgets) {
+            this.removeWidget(widget);
+        }
+
+        this.poiManagerWidgets.clear();
+
+        for (int i = 0, lim = Math.min(pageHeight, waypoints.size() - pageHeight * page); i < lim; i++) {
+            CustomPoi poi = waypoints.get(page * pageHeight + i);
+
+            if (poi == null) {
+                continue;
+            }
+
+            PoiManagerWidget newWidget = new PoiManagerWidget(
```

Comment:
This is the only part I'm unsure of if it's done well, as far as I can tell the widget renders fine but I would prefer it with actual values.

I did try inputting what I thought was correct but it then ended up moving each row to the very left of the window but the Y value was correct so seemingly the X was wrong which I was trying with `this.width/2 - 151` which is the top left position of the icon as that's the leftmost part of the widget.

I've tried resizing my window and different GUI scales and it looks good so it doesn't seem to be a problem but if there's a scenario where it would be a problem then I'm not certain on how to do this correctly